tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "./.svelte-kit/tsconfig.json",
 	"compilerOptions": {
+		"target": "ESNext",
```

Comment:
This was because TS was complaining about some of the mocks being invalid for anything earlier than ES2020 (?), so I specified the latest.